.rubocop_todo.yml

```diff
@@ -68,5 +69,6 @@ RSpec/VerifiedDoubles:
     - 'spec/controllers/media_controller_spec.rb'
     - 'spec/lib/virus_scanner_spec.rb'
     - 'spec/models/asset_spec.rb'
+    - 'spec/workers/virus_scan_worker_spec.rb'
```

Comment:
Very minor: Do we definitely still need the exceptions for `spec/models/asset_spec.rb` in these two lists? i.e. are there still Rubocop violations in the `Asset` spec?

Reply:
Yes, there's still unrelated violations for these exceptions even with the virus related blocks moved across. I'll see if I can address those in a separate PR.